packages/cf-component-input/src/Input.js

```diff
@@ -19,14 +19,7 @@ class Input extends React.Component {
     }
 
     return (
-      <input
-        type={this.props.type}
-        className={className}
-        name={this.props.name}
-        value={this.props.value}
-        onChange={this.handleChange}
-        placeholder={this.props.placeholder}
-        autoComplete={this.props.autoComplete}/>
+      <input className={className} onChange={this.handleChange} {...this.props} />
```

Comment:
@prestonp The handleChange issue is what I was alluring in the ticket. We need to deal with existing whitelisted props on a case by case basis. I think if an event handler is supplied externally in the props, it should mean that the user wants the component to use that one instead of the internal one.

Alternatively, the internal event handler should call the externally supplied event handler. However, **when** (before? after? case by case?) we should call it is not clear. Back in the days when `addEventListener` was available, the order is attachment order, and the convention is to write the event handlers in a way that do not interfere with or depend on each other.

Another option is open up the internal event handler for external event handlers to call, and if an event handling prop is supplied, use it and assume that it will call the internal event handler as it see fit. My preference is this option.